edge-modules/api-proxy-module/docker/linux/arm32v7/Dockerfile

```diff
@@ -5,13 +5,10 @@
 
 ARG DIR=.
 
-FROM arm32v7/alpine:3.12.0
+FROM nginx@sha256:088807753a80fae07f41d44e0e723d04f8fd8acee5d74bf929b926c3d90c10ab
```

Comment:
It seems the comment is lost.  Would you explain again why use hash here instead of image tag, e.g. docker pull nginx:1.19.2?

Reply:
We discussed it, I havent found a way to pull the exact commit I want otherwise.
PS : pulling by digest is not a bad practice